st3/lsp_utils/npm_client_handler.py

```diff
@@ -1,11 +1,30 @@
+from .server_npm_resource import ServerNpmResource
+from LSP.plugin.core.typing import Callable, Dict, List, Optional
 import shutil
 import sublime
 
-from LSP.plugin.core.handlers import LanguageHandler
-from LSP.plugin.core.protocol import Response
-from LSP.plugin.core.settings import ClientConfig, read_client_config
-from LSP.plugin.core.typing import Callable, Dict
-from .server_npm_resource import ServerNpmResource
+USE_NEW_API = True
+
+try:
+
+    from LSP.plugin import ClientConfig
```

Comment:
Let's all shake hands and say "The public API for server handlers is in plugin/\_\_init\_\_.py". Then try to keep that list stable. Importing from anywhere else is not guaranteed to be stable.